examples/src/bin/embassy_hello_world.rs

```diff
@@ -1,41 +0,0 @@
-//! embassy hello world
```

Comment:
Let's keep this example, I know it's basic but hello world is nice to have as an example for new users to run with